### DISCUSSION TITLE
Set some debugger is default

### DISCUSSION THREAD

**Discussion by Freed-Wu** [xmake-io/xmake#5781] [Q&A] [answered]
> We can also pass -d parameters, call debugger programs such as gdb/lldb, load the target file for debugging:
> 
> $ xmake run -d
> 
> xmake will use the debugger that comes with the system to load the program.currently it supports: lldb, gdb, windbg, vsjitdebugger, ollydbg and other debuggers.

`xmake f --menu` display the setting of `Set debugger` is `auto`. How can we let
some debugger has higher priority than other debuggers by some config file or environment var?

**Comment by waruqi** [comment]
set_config("debugger", "xxx")

**Reply by Freed-Wu** [reply]
For every project, it must be write to xmake.lua every time? Is there any global config?

**Reply by waruqi** [reply, marked as answer]
xmake g --debugger=xxx
